client/header/activity-panel/panels/inbox/style.scss

```
@@ -0,0 +1,137 @@
+
+
+.woocommerce-inbox-message {
+	position: relative;
+	color: $gray-text;
+	background: $core-grey-light-200;
+	border-radius: 2px;
+	@include font-size( 13 );
+
+	&.message-is-unread {
+		border: 1px solid $light-gray-secondary;
+		background: $studio-white;
+	}
+}
+
+.woocommerce-inbox-message__content {
+	.woocommerce-inbox-message__title {
+		color: $dark-gray-primary;
+		@include font-size( 16 );
+		font-style: normal;
+		line-height: 24px;
```

Nitpick to favor line height ratio over fixed size:

```suggestion
		line-height: 1.5;
```